src/librustc_trans/trans/context.rs

```diff
@@ -735,6 +735,8 @@ impl<'b, 'tcx> CrateContext<'b, 'tcx> {
     }
 }
 
+/// Remember to add all intrinsics here, in librustc_trans/trans/intrinsic.rs,
+/// in librustc_typeck/check/mod.rs, and in libcore/intrinsics.rs
```

Comment:
Hm this particular comment may not be quite accurate as this commit added intrinsics but didn't modify this file at all. In general while I do think these comments are helpful they're typically hundreds of lines away from the definition so they may be pretty unlikely to be seen.